src/Diagnostics/FxCop/System.Runtime.Analyzers/Core/Design/DefineAccessorsForAttributeArguments.Fixer.cs

```diff
@@ -96,6 +100,7 @@ public class DefineAccessorsForAttributeArgumentsFixer : CodeFixProvider
                                                           (editor, propertyDeclaration) =>
                                                           {
                                                               editor.SetGetAccessorStatements(propertyDeclaration, null);
+                                                              editor.SetModifiers(propertyDeclaration, editor.Generator.GetModifiers(propertyDeclaration) - DeclarationModifiers.WriteOnly);
```

Comment:
[minor] The use of "-" for this seems a bit bizzare. Should we have .Without*() methods in addition to existing .With*() methods?